posthog/test/test_demo.py

```diff
@@ -6,13 +8,15 @@ class TestDemo(BaseTest):
     TESTS_API = True
 
     def test_create_demo_data(self):
+        random.seed(900)
+
         self.client.get("/demo")
         demo_team = Team.objects.get(name__icontains="demo")
-        self.assertEqual(Event.objects.count(), 192)
-        self.assertEqual(Person.objects.count(), 100)
-        self.assertEqual(Action.objects.count(), 3)
+        self.assertEqual(Event.objects.count(), 965)
```

Comment:
Maybe out of the scope of this PR, but think it's quite hard to verify these numbers, not sure if this is really helpful to test expected behaviour

Reply:
Agreed. We need to verify _some_ data exists but the numbers are quite arbitrary. Will think of this next time I change the test.